perf-tool/src/monitor.cpp

```diff
@@ -192,7 +192,7 @@ JNIEXPORT void JNICALL MonitorContendedEntered(jvmtiEnv *jvmtiEnv, JNIEnv *env,
         extensionFunctions++; /* move on to the next extension function */
     }
     
-    sendToServer(j.dump());
+    sendToServer(j.dump(), "monitorContendedEvent");
```

Comment:
This should be "monitorContendedEnteredEvent" because we'll also have a "monitorContendedEnterEvent"

Reply:
@mpirvu - done. also added `monitorContendedEnterEvent` for the newly added function. PTAL.